packages/api/src/external/commonwell/proxy/cw-process-request.ts

```diff
@@ -34,8 +32,9 @@ const countParam = "_count";
 /**
  * CW might send `category` but we've seen it as `('34133-9%5E%5E2.16.840.1.113883.6.1')` which is not valid
  * for the FHIR server, and there's nothing on CW's spec about it, so we're not going to support it for now.
+ * CW might also send `_include`, but since we're not using the FHIR server, we will always return DocumentReferences with the subject (Patient) and the contained Organization
  */
-const allowedQueryParams = ["status", "date", "_include", "_summary"];
+const allowedQueryParams = ["status", "date", "_summary"];
```

Comment:
Only now I noticed something: if they don't add any of these, we'll throw 400, and unless there's something that requires those params on the request, I think we shouldn't.

Thoughts?

Reply:
Ya, the only reason for this that I'm seeing is that the `patient.identifier` ends up in the query parameters. But it's already being parsed in the first step of the request processing. So _I think_ it's safe to remove this constraint